src/mlpack/tests/main.cpp

```diff
@@ -22,8 +22,8 @@ int main(int argc, char** argv)
    * each run.  This is good for ensuring that a test's tolerance is sufficient
    * across many different runs.
    */
-  // size_t seed = std::time(NULL);
-  // math::RandomSeed(seed);
+   // size_t seed = std::time(NULL);
+   // mlpack::math::RandomSeed(seed);
```

Comment:
I think this was correctly formatted before?

Reply:
You're right!  Not sure how I missed that.  Fixed in 41756e656.